test/ret_test.exs

```diff
@@ -342,210 +237,149 @@ defmodule RetTest do
           screenshot_owned_file: image_owned_file,
-          model_owned_file: model_owned_file
+          slug: "fake-scene-slug"
         })
 
-      project |> Ecto.Changeset.change(scene_id: scene.scene_id) |> Repo.update!()
-
-      1 = count(Project, target_account)
-      1 = count(Scene, target_account)
-      3 = count(OwnedFile, target_account)
+      project
+      |> Ecto.Changeset.change(scene_id: scene.scene_id)
+      |> Repo.update!()
 
-      assert :ok = Ret.delete_account(admin_account, target_account)
+      1 = count_where(Project, created_by_account_id: target_account.account_id)
+      1 = count_where(Scene, account_id: target_account.account_id)
+      3 = count_where(OwnedFile, account_id: target_account.account_id)
 
-      assert 0 === count(Project, target_account)
-      assert 0 === count(Scene, target_account)
-      assert 0 === count(OwnedFile, target_account)
+      assert :ok === Ret.delete_account(target_account.account_id, admin_account)
+      assert 0 === count_where(Project, created_by_account_id: target_account.account_id)
+      assert 0 === count_where(Scene, account_id: target_account.account_id)
+      assert 0 === count_where(OwnedFile, account_id: target_account.account_id)
     end
 
     test "deletes parent scenes" do
       {:ok, admin_account: admin_account} = create_admin_account("admin")
       target_account = create_account("target")
       other_account = create_account("other")
-
       project = create_project(target_account)
       scene = create_scene(project, target_account)
 
-      other_project =
-        other_account
-        |> create_project()
-        |> Ecto.Changeset.change(parent_scene_id: scene.scene_id)
-        |> Repo.update!()
-
-      other_project
+      other_account
+      |> create_project()
+      |> Ecto.Changeset.change(parent_scene_id: scene.scene_id)
+      |> Repo.update!()
       |> create_scene(other_account)
       |> Ecto.Changeset.change(parent_scene_id: scene.scene_id)
       |> Repo.update!()
 
-      1 = count(Project, target_account)
-      1 = count(Scene, target_account)
-      1 = count(Project, other_account)
-      1 = count(Scene, other_account)
-
-      assert :ok = Ret.delete_account(admin_account, target_account)
+      1 = count_where(Scene, account_id: target_account.account_id)
+      1 = count_where(Scene, account_id: other_account.account_id)
 
-      assert 0 === count(Project, target_account)
-      assert 0 === count(Scene, target_account)
-      assert 1 === count(Project, other_account)
-      assert 1 === count(Scene, other_account)
+      assert :ok === Ret.delete_account(target_account.account_id, admin_account)
+      assert 0 === count_where(Scene, account_id: target_account.account_id)
+      assert 1 === count_where(Scene, account_id: other_account.account_id)
     end
 
-    test "listed scenes are reassigned to admin account" do
+    test "reassigns listed scenes to admin account" do
```

Comment:
Shouldn't it delete the scene listings and scenes that aren't remixed? Not reassign the scenes.

Reply:
@johnshaughnessy @netpro2k Do you have any thoughts?

Reply:
I'm not sure what the correct behavior is. It was clearly an intentional choice (Brian noted it in the original PR https://github.com/mozilla/reticulum/pull/624 ). I'm trying to understand how scene listings are used, and also reached out to Brian to ask why he implemented things this way.

Since it's not an accident and we have enough information to change the implementation later, I think it's safe to ship before we get to the bottom of it, in case it takes us a while.

Reply:
Ok I'm ramped up now. This is ultimately a decision that product should make, because it's about "who owns what" and what it means to allow promotion / featuring of scenes. It is related to remixing. Similar questions come up for avatars. Let's discuss options with Elgin.

Reply:
@bryanenders and I spoke with @emclaren about this topic. Summarizing for @robinkwilson and posterity.

(Disclaimer: I'm not a lawyer.)

Reassigning `scene_listings` (not deleting them) seems to be the desired behavior, since `scene_listings` only exist for scenes that have been marked promote-able, which seems to give Mozilla permission: https://github.com/mozilla/Spoke/blob/master/PROMOTION.md 

Similarly, marking an asset as "remixable" comes with certain terms: https://github.com/mozilla/Spoke/blob/master/REMIXING.md

This means that:
- Alice publishes a scene. Marks it promotable.
- Bob (hub owner) approves / features it, which creates a scene listing.
- Charlie creates a room with that scene listing.
- Alice deletes her account. This deletes the scene, but not the listing. Ownership of the underlying asset (Owned file) is (lazily) transferred to Bob.
- Charlie's room doesn't change. 

We may want to revisit this behavior in the future and make sure users are totally aware of who is allowed to do what with their assets. As Elgin recalled, once-upon-a-time we had these "remixable" and "promotable" options on by default, and we changed the defaults to be disabled to make it less likely to happen on accident. We may consider building a formal mechanism for users to ask the hub owner to un-publish an asset.

We should also make sure the copy is correct for individual hubs. The links above say that _Mozilla_ is given rights to do things, but I believe these capabilities are conferred on _the hub admin_, who is usually _not_ Mozilla.

Reply:
> The links above say that Mozilla is given rights to do things, but I believe these capabilities are conferred on the hub admin, who is usually not Mozilla.

We will need to decide. It might be both for our hosted service.